conans/client/conan_command_output.py

```diff
@@ -133,16 +132,16 @@ def _grab_info_data(self, deps_graph, grab_paths):
 
             # Paths
             if isinstance(ref, ConanFileReference) and grab_paths:
-                item_data["export_folder"] = self.cache.export(ref)
-                item_data["source_folder"] = self.cache.source(ref, conanfile.short_paths)
-                if isinstance(self.cache, SimplePaths):
-                    # @todo: check if this is correct or if it must always be package_id
-                    package_id = build_id(conanfile) or package_id
-                    pref = PackageReference(ref, package_id)
-                    item_data["build_folder"] = self.cache.build(pref, conanfile.short_paths)
-
-                    pref = PackageReference(ref, package_id)
-                    item_data["package_folder"] = self.cache.package(pref, conanfile.short_paths)
+                package_layout = self.cache.package_layout(ref, conanfile.short_paths)
```

Comment:
Why if you are introducing "shortcut" methods in the local cache to access to the export folder then you change to use the package_layout? should we better remove the shortcut methods?

Reply:
Yes, the goal is to remove the methods from the cache. It just would be too much to do it in one PR, because they are still heavily used in tests, and other places we cannot still remove them. So we need to go step by step, do it whenever possible.

Reply:
And by the way, I am not introducing them. They were there, just inherited from ``SimplePaths``, which ``ClientCache`` was extending. I have now made it more evident.